react/week1/my-react-app/src/todo.css

```diff
@@ -0,0 +1,21 @@
+.todoitem{
```

Comment:
once more, since the app is very small you could put this styles on App.css, no need of another file. 😉

Reply:
Ok @annamourasobral , I will update :)